compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -124,7 +130,6 @@ fn rustc_op_to_op(op: &rustc_middle::mir::Operand<'_>) -> stable_mir::mir::Opera
 }
 
 fn rustc_place_to_place(place: &rustc_middle::mir::Place<'_>) -> stable_mir::mir::Place {
-    assert_eq!(&place.projection[..], &[]);
```

Comment:
Removing this assert is wrong unless you lower the projection

Reply:
You could just debug format it for now and add the string

Reply:
I was getting this error:

```
thread 'rustc' panicked at 'assertion failed: `(left == right)`
  left: `[Field(1, bool)]`,
 right: `[]`', compiler/rustc_smir/src/rustc_smir/mod.rs:133:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I'm not sure what did you mean by just debug format it for now and add the string?

Reply:
Add a `projection: String` field and use `format!("{:?}", place.projection)`. As long as it's clear that two values are not the same, it doesn't matter how badly we represent them. This is what I did for types, and you can just use it in places where you'd have to do a lot of work to support sth

Reply:
Done!